package.json

```diff
@@ -49,18 +49,17 @@
     "karma-firefox-launcher": "^1.3.0",
     "karma-mocha": "^1.3.0",
     "karma-mocha-reporter": "^2.2.5",
-    "karma-sauce-launcher": "^1.2.0",
+    "karma-sauce-launcher": "^2.0.2",
```

Comment:
major version bumps 💪

Reply:
Didn't aligned on v4 because it is broken (https://github.com/karma-runner/karma-sauce-launcher/issues/190)